Troubleshooting.md

```diff
@@ -688,6 +690,21 @@ Following the completion of the aforementioned commands, you can proceed with th
 
 [Reference](https://stackoverflow.com/questions/72299444/docker-desktop-doesnt-install-saying-docker-ce-cli-not-installable) for the solution stated.
 
+### make commands: `Operation not permitted`
+
+Some users might face the following error while running `make` commands:
+```
+Operation not permitted
+make: *** [Makefile:14: docker] Error 1
+```
+
+The simple solution for the above error is to run the make commands with admin access i.e. with `sudo`, example:
```

Comment:
Users will likely want to avoid running sudo where possible, because this gives the script elevated permissions. Can we instead fix things so that they don't need to do this? E.g. https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo

Reply:
done. PTAL